Tests/SwiftTestingTestableCombinePublishersTests/SwiftTestingTestableCombinePublishersTests.swift

```diff
@@ -0,0 +1,525 @@
+//
+//  SwiftTestingTestableCombinePublishersTests.swift
+//  TestableCombinePublishers
+//
+//  Created by Ethan van Heerden on 11/15/24.
+//
+
+import Testing
+import Combine
+import Foundation
+@testable import SwiftTestingTestableCombinePublishers
+
+final class SwiftTestingTestableCombinePublishersTests {
+    
+    // MARK: - Receive Value Expectations
+    
+    @Test func testExpectEquatableValue() async {
+        await ["cool"]
+            .publisher
+            .expect("cool")
+            .waitForExpectations(timeout: 1)
+        
+        // TODO: I had to add .collect() logic here
```

Comment:
req: Let's get to the bottom of this so that we can ensure 1:1 with the previous design. It might be because this expectation get "over-fulfilled" by matching all 3 values emitted. (See [this line](https://github.com/albertbori/TestableCombinePublishers/pull/6/files?diff=unified&w=0#diff-8ec611b09b8f6186caafbcbf42630788c7a60a70e324c4ca18e5981bb16ff445R37))

Reply:
Yes you are right, it is caused by that line. Changing the implementation of `isFulfilled` to be:
```swift
var isFulfilled: Bool {
        return actualFulfillmentCount >= expectedFulfillmentCount
    }
```

fixes the issue and behaves like the default `XCTestExpectation` implementation. Do you think we should add support for the [`assertForOverFulfill`](assertForOverFulfill) XCTest counterpart in the Swift Testing implementation?